docs/mkdocs/developer/index.md

```diff
@@ -70,29 +73,31 @@ status:
 ```
 
 ## In-cluster builds
+
 Virtually any Kubernetes distribution ships with its own kernel.
 It is thus challenging for a kernel module vendor to make DriverContainer images available for all kernels available
 in all Kubernetes distributions out there (or even for a subset of them).
 
-KMM supports in-cluster builds of DriverContainer images when those are not made available by the vendor.
-The in-cluster build system is able to build an image from any Git repository that contains a `Dockerfile`.
-It can then push said image to a user-provided repository, optionally using authentication provided in a Kubernetes secret.
+KMM supports in-cluster builds of DriverContainer images when those are not made available by the vendor. The in-cluster build system is able to build an image from any Git repository that contains a `Dockerfile`. It can then push said image to a user-provided repository, optionally using authentication provided in a Kubernetes secret.
+
+**Optional:** on upstream Kubernetes, we may want to deploy an in-cluster registry to host in-cluster built images.
 
-**Optional:** on upstream Kubernetes, we may want to deploy an in-cluster registry to host in-cluster built images.  
 **On OCP**, the build mechanism would be BuildConfig (maybe Shipwright in the future) and we can leverage the
 integrated in-cluster registry.
```

Comment:
I would also add here that the internal registry isn't enabled by default on bare metal installations and needs to be enabled manually:  https://docs.openshift.com/container-platform/4.11/registry/configuring_registry_storage/configuring-registry-storage-baremetal.html